### PR TITLE
[Release 10.3.0] Prepare Changelog for RC1

### PR DESCRIPTION
**Pull request by micbar**
## Description
Prepare the changelog for RC1

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Review by phil-davis — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,9 +23,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - `previews_path` config option to configure thumbnail storage path [#35131](https://github.com/owncloud/core/pull/35131)
 - Show activity when share receiver unshares a received share [#35193](https://github.com/owncloud/core/pull/35193)
 - Document phoenix.baseUrl in config.sample.php - [#36007](https://github.com/owncloud/core/issues/36007)
+- Add getReshareAttributes method to shareitemmodel with fix for parsing - [#36186](https://github.com/owncloud/core/issues/36186)
+- Add new migrations to dav app to prevent invalid dav properties - [#36084](https://github.com/owncloud/core/issues/36084)
 
 ### Changed
 
+- Disable Tech preview trashbin and public DAV APIs by default - [#36124](https://github.com/owncloud/core/issues/36124)
```
Comment (PVince81) [inline CHANGELOG.md]: this is not a change from a previous release.

let's move it to "Added".

File: CHANGELOG.md [hunk 2]
```diff
@@ -67,6 +74,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Fixed
 
+- Improve share_folder checks in rmdir - [#36170](https://github.com/owncloud/core/issues/36170)
```
Comment (PVince81) [inline CHANGELOG.md]: would be good to have a less technical description

maybe "Fix potential issue when a user tries to delete the share_folder entry"

File: CHANGELOG.md [hunk 3]
```diff
@@ -67,6 +74,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
+- Fix invalid foreach in SharingBlacklist.php - [#36038](https://github.com/owncloud/core/issues/36038)
```
Comment (PVince81) [inline CHANGELOG.md]: this is more tech debt, code cleanup

File: CHANGELOG.md [hunk 4]
```diff
@@ -67,6 +74,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
+- obey to config in share mail notifications APIs - [#36161](https://github.com/owncloud/core/issues/36161)
+- Don't invalidate the auth token if there isn't a user session active - [#36153](https://github.com/owncloud/core/issues/36153)
+- Fix typos in 'phoenix.baseUrl' documentation - [#36152](https://github.com/owncloud/core/issues/36152)
+- Don't check the CSRF token on public link email API - [#36158](https://github.com/owncloud/core/issues/36158)
+- Remove hardcoded http response codes. - [#36127](https://github.com/owncloud/core/issues/36127)
```
Comment (PVince81) [inline CHANGELOG.md]: remove extra dot

File: CHANGELOG.md [hunk 5]
```diff
@@ -67,6 +74,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
+- Share owner can edit re-share permissions without any restriction - [#36193](https://github.com/owncloud/core/issues/36193)
```
Comment (PVince81) [inline CHANGELOG.md]: "Share owner can now edit ..." as it's the new behavior after the fix

**Review by PVince81 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 5 above)
Comment (phil-davis) [reply CHANGELOG.md]: I think this is another example of a between-releases regression that was fixed without the regression ever being released. We did the permissions refactoring, and then discovered this regression and fixed it. IMO it is not a problem that is in 10.2.1 (?)

File: CHANGELOG.md (hunk 1 above)
Comment (micbar) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 3 above)
Comment (micbar) [reply CHANGELOG.md]: remove?

File: CHANGELOG.md (hunk 5 above)
Comment (micbar) [reply CHANGELOG.md]: Fix permission handling for share owner of a reshare

File: CHANGELOG.md (hunk 3 above)
Comment (PVince81) [reply CHANGELOG.md]: rename to "Clean up code of sharing blacklist feature"

File: CHANGELOG.md (hunk 5 above)
Comment (PVince81) [reply CHANGELOG.md]: sounds good

File: CHANGELOG.md [hunk 6]
```diff
@@ -67,6 +74,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Fixed
 
+- Fix potential issue when a user tries to delete the share_folder entry - [#36170](https://github.com/owncloud/core/issues/36170)
+- Clean up code of sharing blacklist feature - [#36038](https://github.com/owncloud/core/issues/36038)
+- obey to config in share mail notifications APIs - [#36161](https://github.com/owncloud/core/issues/36161)
```
Comment (PVince81) [inline CHANGELOG.md]: Capitalize "O"

**Review by PVince81 — APPROVED** [review]
other than that :+1:

**Pull request merged**